webservices/args.py

```diff
@@ -412,11 +412,18 @@ def make_seek_args(field=fields.Int, description=None):
         validate=validate.OneOf(['house', 'senate', 'president']),
         description='Office sought, either President, House or Senate.',
     ),
+    'period': fields.Bool(missing=False),
 }
 
 schedule_a_candidate_aggregate = {
     'candidate_id': fields.List(IStr, required=True, description=docs.CANDIDATE_ID),
     'cycle': fields.List(fields.Int, required=True, description=docs.RECORD_CYCLE),
+    'period': fields.Bool(missing=False),
```

Comment:
I'm not quite sure I fully understand the meaning of this argument... can you explain?